src/openms/source/ANALYSIS/ID/SiriusAdapterAlgorithm.cpp

```diff
@@ -183,7 +184,31 @@ namespace OpenMS
     {
       return tmp_ms_file_;
     }
-    
+
+    std::vector<String> SiriusAdapterAlgorithm::sortSiriusWorkspacePathsByScanIndex(const std::vector<String>& siriusworkspacepaths)
```

Comment:
@cbielow I think that should go in the right direction - do you have further suggestions?